homeassistant/components/myq/cover.py

```diff
@@ -106,23 +74,23 @@ async def async_close_cover(self, **kwargs):
         try:
             wait_task = await self._device.close(wait_for_state=False)
         except MyQError as err:
-            _LOGGER.error(
-                "Closing of cover %s failed with error: %s", self._device.name, str(err)
-            )
-
-            return
+            # Write current state which would be Open to HASS
+            self.async_write_ha_state()
```

Comment:
You should not write the state here (This also seems unrelated to creating a MyQEntity), did you mix your branches?

Reply:
Already removed. :-) This is from another PR (hence I put this one as a draft PR).